### DISCUSSION TITLE
use a self-built domain email for registration verification, the link to verify the email jumps to localhost

### DISCUSSION THREAD

**Discussion by harrisonhxy** [danny-avila/LibreChat#4837] [Troubleshooting]
### What happened?

I am using a self-built domain for registration verification, and the registered users have successfully received the email, but the link for **verify email** to click on ishttp://localhost:端口/verify?token=xxxxxxxxxxxxxxxx=Registered user email
<img width="788" alt="image" src="https://github.com/user-attachments/assets/26295bdf-cafe-4aba-82bf-6b391425c128">
<img width="1067" alt="image" src="https://github.com/user-attachments/assets/b43f8a87-ddfa-42fa-8a60-78298aba2005">

### Steps to Reproduce

1.Reference documents for use
<img width="861" alt="image" src="https://github.com/user-attachments/assets/bde1c50f-66a3-404c-9aff-9fd8ded0d9cb">

2.The email settings in my .env environment variable are as follows
<img width="364" alt="image" src="https://github.com/user-attachments/assets/b2100115-68b1-41cd-b091-8400d8024ffe">

3.The log content shows that the registration connection was successfully sent
<img width="1294" alt="image" src="https://github.com/user-attachments/assets/d3797e1b-b9ca-42b0-949e-d4ff692c6456">

4.Due to the jump to **localhost**, I had to change localhost to my own server IP, and then the registration was successful.
<img width="1192" alt="image" src="https://github.com/user-attachments/assets/63bf2200-8ccd-4cf4-a27b-1a6e58806b59">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Update the `DOMAIN_CLIENT` in the `.env` file to your domain

@fuegovic @berry-13 this should be added to the docs: https://www.librechat.ai/docs/configuration/authentication/email
